Library/Homebrew/cmd/help.rb

```diff
@@ -74,6 +74,12 @@ def help(cmd = nil, flags = {})
   end
 
   def command_help(path)
+    if Pathname(path).fnmatch?("*/Homebrew/cmd/*.rb")
+      cmd = Pathname(path).basename(".rb")
+      class_name = "#{cmd.to_s.capitalize}Command"
```

Comment:
How are commands containing `-` and other characters handled? For example, `gist-logs`, `tap-info`, or `--cellar`?